recipes/bitserializer/0.10/conanfile.py

```diff
@@ -51,7 +52,7 @@ def requirements(self):
         if self.options.with_cpprestsdk:
             self.requires("cpprestsdk/2.10.16")
         if self.options.with_rapidjson:
-            self.requires("rapidjson/1.1.0")
+            self.requires("rapidjson/cci.20200410")
```

Comment:
I think it's better if the default was an official release :thinking:

Reply:
agree

Reply:
ok, bump cancelled